test/common/router/rds_impl_test.cc

```diff
@@ -229,6 +230,7 @@ TEST_F(RdsImplTest, Basic) {
 
   // Test Admin /routes handler. Now we should have an empty route table, with exception of name.
   const std::string routes_expected_output_only_name = R"EOF({
+    "version_info": "hash_5155986269879860237",
```

Comment:
Optimally, you would have a test where we actually setup the gRPC fetcher, and make sure the version_info output is what is sent via API. I would at least put in a TODO around that.

Reply:
Seems that `rds_impl_test.cc` is concerned with consuming version info, rather than correctly tracking it across different APIs (which is what is going on in the `common/config/*_impl_test`).